src/OrchardCore.Modules/Orchard.OpenId/Startup.cs

```diff
@@ -42,6 +42,7 @@ public class Startup : StartupBase
 
         public override void Configure(IApplicationBuilder builder, IRouteBuilder routes, IServiceProvider serviceProvider)
         {
+            /*
```

Comment:
Hum, this makes the OpenID module completely unusable. I'll try to take a look before you merge it.